apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.spec.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
```

Comment:
The diff is hard to read. I extracted the render method to a function `renderSignedVerdictOverview` and added some tests.